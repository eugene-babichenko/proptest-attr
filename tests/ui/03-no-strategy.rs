use proptest_attr::proptest;

fn main() {
    basic_test();
}

#[proptest]
fn basic_test(_value: u8) -> Result<(), TestCaseError> {
    Ok(())
}
