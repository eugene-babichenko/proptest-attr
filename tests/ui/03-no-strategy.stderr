error: no strategy specified for this proptest
 --> $DIR/03-no-strategy.rs:7:1
  |
7 | #[proptest]
  | ^^^^^^^^^^^
  |
  = note: this error originates in an attribute macro (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0425]: cannot find function `basic_test` in this scope
 --> $DIR/03-no-strategy.rs:4:5
  |
4 |     basic_test();
  |     ^^^^^^^^^^ not found in this scope
